### PR TITLE
Func-10-12

### PR DESCRIPTION
**Pull request by yalytsia**

### CODE REVIEW

**Comment by VsIG-official** [comment]
EditBooking.png looks good, but maybe We need to write "system validate..." more detailly. For example, user edit booking and want another hotel "B", but while user is editing, this hotel is deleted.
Though I don't know, if We need to write situations like this into diagram

**Comment by VsIG-official** [comment]
EditBooking.png looks good, but maybe We need to write "system validate..." more detailly.
For example, user want to change hotel "A" to hotel "B", but while the user is editing, this hotel is gone.
Though I don't know, if We need to write situations like this into diagram

**Comment by VsIG-official** [comment]
LogOut.png is fine for Me. I just don't know, what kind of problems may occur, while logging out

**Comment by VsIG-official** [comment]
In ReportComment.png maybe We need to specify what type of "bad" content is in comment.
For example "Aggresive","call to suicide"* and etc.
*Well, that's just an example, this kind of report is not necessary

File: docs/EditBooking.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+|Authorized user|
+start
+:User presses button "Edit" on booking;
+|System|
+:System opens page 
+"Edit booking details";
+|Authorized user|
+:User makes changes and submits them;
+|System|
+:System validates and update DB;
```
Comment (VsIG-official) [inline docs/EditBooking.txt:23]: first commentary from Me

File: docs/ReportComment.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<font color=000 size=16><b>Main scenario:</b>
+
+end header
+|Authorized user|
+start
+:User presses button "Report comment" on a comment;
+|System|
+:System opens "Report options" dialog window; 
+|Authorized user|
+:User chooses report option;
```
Comment (VsIG-official) [inline docs/ReportComment.txt:19]: Third comment from Me

**Review by VsIG-official — COMMENTED** [review]
Just My thoughts, that may not affect the end result

File: docs/EditBooking.txt (hunk 1 above)
Comment (fhrr-sht) [reply docs/EditBooking.txt:23]: I don't think we should specify all details in our diagrams. 
There are many options in validation, we need to discuss them with the team.
For example, can we edit past date (to edit orders of previous years) etc.

File: docs/ReportComment.txt (hunk 2 above)
Comment (fhrr-sht) [reply docs/ReportComment.txt:19]: "User chooses report option" -> User can choose "Aggresive","call to suicide"* etc to specify the reason to report a comment. Diagram is a high level representation of our system. Also, I am sure that we change details a few times.

**Review by La7rodectus — APPROVED** [review]
Good.

**Pull request merged**